pkg/client/docs/DefaultApi.md

```diff
@@ -196,6 +197,82 @@ Name | Type | Description  | Notes
 [[Back to README]](../README.md)
 
 
+## CreateVolumePoolSizeNameGet
+
+> VolumesObject CreateVolumePoolSizeNameGet(ctx, poolOption, sizeOption, nameOption).Execute()
+
+
+
+
+
+### Example
+
+```go
+package main
+
+import (
+    "context"
+    "fmt"
+    "os"
+    openapiclient "github.com/GIT_USER_ID/GIT_REPO_ID"
+)
+
+func main() {
+    poolOption := "poolOption_example" // string | 
```

Comment:
What's the "|" for in the comment?

Reply:
This is auto-generated documentation from openAPI, I am not sure what the reasoning is for the '|' characters.